src/API/HealthController.ts

```diff
@@ -1,36 +1,48 @@
 import { Db, Collection } from 'mongodb'
+import * as Pino from 'pino'
+
+import { TransactionAnchorRetryInfo } from 'Interfaces'
 
 export const isOkOne = ({ ok }: { ok: number }) => ok === 1
 
-interface HealthObject {
+export interface HealthObject {
   readonly mongoIsConnected: boolean
   readonly ipfsInfo: object
   readonly walletInfo: object
   readonly blockchainInfo: object
   readonly networkInfo: object
   readonly estimatedSmartFeeInfo: object
   readonly ipfsRetryInfo: object
+  readonly transactionAnchorRetryInfo: TransactionAnchorRetryInfo
 }
 
 export interface Dependencies {
   readonly db: Db
+  readonly logger: Pino.Logger
 }
 
 export interface Arguments {
   readonly dependencies: Dependencies
 }
 
+const emptyTransactionAnchorRetryInfo: { transactionAnchorRetryInfo: TransactionAnchorRetryInfo } = {
```

Comment:
formatting this line to be a bit shorter would be nice.

Reply:
should we modify our ts-lint rules to enforce shorter lines?

Reply:
I created an interface to shorten the line

Reply:
lol :)

I think prettier originally kept our lines short, looks like it when it was removed we increased it to 120 https://github.com/poetapp/tslint-rules/blame/b0b654f6419a5a9c75f8a0991ecabe42186c71ee/poet-tslint.json#L33

> I created an interface to shorten the line

Thanks :D

Reply:
We should reduce it if prettier was keeping it shorter